### DISCUSSION TITLE
Straight edge rotating 360

### DISCUSSION THREAD

**Discussion by the-mgi** [xyflow/xyflow#2414] [Q&A]
is it possible to have a straight line rotate around the target edge node as the curved one is moving? here are the two sample videos

any help would highly be appreciated.
thanks 🙌🏼 

[Screencast from 08-09-2022 15:05:31.webm](https://user-images.githubusercontent.com/70032816/189096057-08a6db33-9e68-42bc-9687-dcb7f1bcd73d.webm)
[Screencast from 08-09-2022 15:06:20.webm](https://user-images.githubusercontent.com/70032816/189096064-20b16be3-f7aa-45bf-866c-503e206faa36.webm)

**Comment by moklick** [comment]
Do you want the behaviour of the floating edge with a straight edge?

**Reply by the-mgi** [reply]
yeah you can say that, because the arrow gets hidden underneath the node so that's why
